e2e/capacity/staking.test.ts

```diff
@@ -60,7 +54,7 @@ describe("Capacity Staking Tests", function () {
 
         it("successfully unstakes the minimum amount", async function () {
             const stakeObj = ExtrinsicHelper.unstake(stakeKeys, stakeProviderId, tokenMinStake);
-            const [unStakeEvent] = await stakeObj.fundAndSend();
+            const [unStakeEvent] = await stakeObj.fundAndSend(fundingSource);
```

Comment:
Good example of a lot of the changed lines. Effectively giving the funding source here instead of at the root